doc/classes/Image.xml

```diff
@@ -130,6 +134,7 @@
 			<argument index="4" name="data" type="PoolByteArray">
 			</argument>
 			<description>
+				Creates a new image with specified size and format. Fills the image with the raw data provided.
```

Comment:
- see comments on `create` above
- "with the raw data provided" -> with the given raw data